QUEST_LV_0300.tsv

```diff
@@ -209,91 +209,91 @@ QUEST_LV_0300_20150908_000208	You've failed to collect pertification powder!
-QUEST_LV_0300_20151001_000233	Demon Queen Gesti brought the curse of the petrification on this land,{nl}and the goddesses tried to save the humans. But, the demons asked us whether {nl}we have forgotten not to interfere with each other's matters and...{nl}
+QUEST_LV_0300_20151001_000232	But, the one who broke it first were the demons.{nl}That is this petrified frost... they died and their spirits have become stone.{nl}
+QUEST_LV_0300_20151001_000233	Demon Queen Gesti brought the curse of the petrification on this land,{nl}and the goddesses tried to save the humans. But, the demons asked us whether {nl}we had forgotten not to interfere with each other's matters and...{nl}
 QUEST_LV_0300_20151001_000234	Wait. This memory is too horrible..{nl}A horrible catatrophe has landed.{nl}
-QUEST_LV_0300_20151001_000235	Like a big disaster that the goddess might abandon the save this petrified place.{nl}These Stone Frosts... All of that... This is what we pay for the save that we abandon.
-QUEST_LV_0300_20151001_000236	I am trying to save the spirits from now on.{nl}Since you fixed all the detectors, we would have to guide the spirits to eternal sleep.
+QUEST_LV_0300_20151001_000235	Like a big disaster that the goddess might abandon to save this petrified place.{nl}These Stone Frosts... All of that... This is what we must suffer through to save that which we abandoned.
+QUEST_LV_0300_20151001_000236	I will try to save the spirits from now on.{nl}Since you fixed all the detectors, we will be able to guide the spirits to eternal sleep.
 QUEST_LV_0300_20151001_000237	We will do our best to guide each and every one of them.{nl}I hope the goddesses bless you.
 QUEST_LV_0300_20151001_000238	I guess you can't trust me since a demon like me is asking you a favor.{nl}But, look at me closely. It doesn't matter if you laugh at me.{nl}
-QUEST_LV_0300_20151001_000239	I've tried my best to escape for hundreds of years continuously.{nl}But, the curse of the petrification is the power of the Demon Queen. {nl}The endless depression is suffocating me.{nl}
-QUEST_LV_0300_20151001_000240	Even if you escape from here, you are just betrayer. You will get the word "betrayer". {nl}While going to the feet of the colleagues who mocked you...{nl}
-QUEST_LV_0300_20151001_000241	If you cooperate on my death, I will definitely remove your curse.{nl}I promise in the name of Giltine.
-QUEST_LV_0300_20151001_000242	It doesn't matter what happens to my body.{nl}I just want to take some rest...
+QUEST_LV_0300_20151001_000239	I've tried my best to escape for hundreds of years.{nl}But, the curse of the petrification is the power of the Demon Queen. {nl}This endless depression is suffocating me.{nl}
+QUEST_LV_0300_20151001_000240	Even if you escape from here, you are just a betrayer. You will get the word "betrayer". {nl}While going to the feet of the colleagues who mocked you...{nl}
```

Comment:
"Even if you escape from here, you are just a betrayer. You will get the word "betrayer". {nl}While going to the feet of the colleagues who mocked you...{nl}"
> Even if you escape, you're still a traitor.{nl}You'll be called a "traitor" while at the feet of the colleagues who mocked you...{nl}